go/oasis-test-runner/scenario/e2e/txsource.go

```diff
@@ -270,6 +284,7 @@ func (sc *txSourceImpl) startWorkload(childEnv *env.Env, errCh chan error, name
 		"--" + workload.CfgRuntimeID, runtimeID.String(),
 		"--" + txsource.CfgWorkload, name,
 		"--" + txsource.CfgTimeLimit, sc.timeLimit.String(),
+		"--" + txsource.CfgSeed, sc.seed,
```

Comment:
how did the txsource process used to get the seed?

Reply:
used the default https://github.com/oasislabs/oasis-core/blob/master/go/oasis-node/cmd/debug/txsource/txsource.go#L136